<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="ComLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect to COM:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="ComSpinBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>42</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ComButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="LedLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>151</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: black;</string>
    </property>
    <property name="text">
     <string>LED</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LedButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>On/Off</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>101</y>
      <width>256</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>688</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuVirtual_LED">
    <property name="title">
     <string>Virtual LED</string>
    </property>
   </widget>
   <addaction name="menuVirtual_LED"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actiong">
   <property name="text">
    <string>g</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
